src/Elasticsearch.Net/Serialization/Formatters/ExceptionFormatter.cs

```diff
@@ -3,30 +3,32 @@
 using System.Globalization;
 using System.Reflection;
 using System.Runtime.Serialization;
+using Elasticsearch.Net.Extensions;
 using Elasticsearch.Net.Utf8Json;
 
 namespace Elasticsearch.Net
 {
 	internal class ExceptionFormatterResolver : IJsonFormatterResolver
 	{
-		public static ExceptionFormatterResolver Instance = new ExceptionFormatterResolver();
+		public static readonly ExceptionFormatterResolver Instance = new ExceptionFormatterResolver();
 
 		private ExceptionFormatterResolver() { }
 
-		private static readonly ExceptionFormatter ExceptionFormatter = new ExceptionFormatter();
-
 		public IJsonFormatter<T> GetFormatter<T>()
 		{
 			if (typeof(Exception).IsAssignableFrom(typeof(T)))
-				return (IJsonFormatter<T>)ExceptionFormatter;
+			{
+				var type = typeof(ExceptionFormatter<>).MakeGenericType(typeof(T));
```

Comment:
`.CreateGenericInstance()`

Reply:
`CreateGenericInstance` exists in Nest, but `ExceptionFormatter` is in `Elasticsearch.Net`. Can move the extension method to `Elasticsearch.Net`?

Reply:
I think we can, we can also follow up in a new issue.